templates/taxi/car_list.html

```diff
@@ -1,22 +1,27 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>
-    Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
-  </h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
+	<h1>
+		Car list
+		<a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
+			Create
+		</a>
+	</h1>
+	<form method="get" action="">
+		{{ search_form|crispy }}
+		<input type="submit" value="Search" class="btn btn-primary">
+	</form>
+	{% if car_list %}
+		<ul>
+			{% for car in car_list %}
+				<li>
+					<a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
The URL tag for the car detail link uses double quotes incorrectly. Change `"taxi:car-detail"` to `'taxi:car-detail'` to ensure proper syntax.